src/sharedHelpers/taxon.ts

```diff
@@ -101,8 +97,24 @@ export const capitalizeCommonName = name => {
   } ).join( " " );
 };
 
-export const generateTaxonPieces = taxon => {
-  const taxonData = {};
+interface Taxon {
+  rank?: string;
+  rank_level: number;
+  preferred_common_name?: string;
+  name?: string;
+}
+
+interface TaxonData {
```

Comment:
This name is a little confusing for me now that it's written into TypeScript -- can we think of a way to name this to differentiate why this is needed vs. why `Taxon` is needed? Maybe this can be called something like `TaxonDisplayData` and/or `Taxon` can be `ApiTaxon`? I'm not sure if we have a naming convention for this already, but it's nice to understand what we've transformed versus what is taken directly from the API.